cluster/kubernetes/manifests.go

```diff
@@ -60,31 +74,48 @@ func getCRDScopes(manifests map[string]kresource.KubeManifest) ResourceScopes {
 	return result
 }
 
-func setEffectiveNamespaces(manifests map[string]kresource.KubeManifest, nser namespacer) (map[string]resource.Resource, error) {
+func (m *manifests) setEffectiveNamespaces(manifests map[string]kresource.KubeManifest) (map[string]resource.Resource, error) {
 	knownScopes := getCRDScopes(manifests)
 	result := map[string]resource.Resource{}
 	for _, km := range manifests {
-		if nser != nil {
```

Comment:
I am pretty sure this check was here for the tests, so I simply made sure that we never used a `nil` namespacer.

Reply:
Tried to confirm this by going back through history but it has seen too many shapes in the past weeks for me to get a clear view. I will trust your 'pretty sure' on this one as you have been living in this part of Flux for quite a while.